src/ucp/tag/eager_rcv.c

```diff
@@ -284,9 +283,8 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_eager_sync_ack_handler,
     if (worker->context->config.ext.proto_enable) {
         ucp_proto_eager_sync_ack_handler(worker, rep_hdr);
     } else {
-        UCP_WORKER_EXTRACT_REQUEST_BY_ID(&req, worker, rep_hdr->req_id,
-                                         return UCS_OK, "EAGER_S ACK %p",
-                                         rep_hdr);
+        UCP_REQUEST_GET_BY_ID(&req, worker, rep_hdr->req_id, 0, return UCS_OK,
```

Comment:
extract?

Reply:
we should delete request ID after all requests are sent
otherwise - getting local request ID will be broken, since it is set to `INVALID`